201-web-app-asev2-create/azuredeploy.parameters.json

```diff
@@ -1,21 +1,18 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json",
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json",
     "contentVersion": "1.0.0.0",
     "parameters": {
         "aseName": {
-            "value": "GEN_UNIQUE"
+            "value": "GEN-UNIQUE"
         },
         "aseLocation": {
-            "value": "South Central US"
+            "value": "changeme"

```

Comment:
use a valid location

Reply:
This field needs to be the same location that the VNet was created in that the ASE is deployed into.  Why put in a specific location?

Reply:
so we can run ci/cd and test the common scenario